tests/garage/experiment/test_snapshotter.py

```diff
@@ -19,41 +24,78 @@
 }), ('none', {})]
 
 
-class TestSnapshotter:
+def sample_task(snapshot_config):
+    with LocalRunner(snapshot_config=snapshot_config) as runner:
+        env = TfEnv(env_name='CartPole-v1')
+
+        policy = CategoricalMLPPolicy(
+            name='policy', env_spec=env.spec, hidden_sizes=(32, 32))
+
+        baseline = LinearFeatureBaseline(env_spec=env.spec)
+
+        algo = VPG(
+            env_spec=env.spec,
+            policy=policy,
+            baseline=baseline,
+            max_path_length=100,
+            discount=0.99,
+            optimizer_args=dict(tf_optimizer_args=dict(learning_rate=0.01, )))
+
+        runner.setup(algo, env)
+        runner.train(n_epochs=5, batch_size=10)
+
+
+class TestSnapshotter(TfGraphTestCase):
```

Comment:
please write this as a unit test which doesn't rely on classes outside Snapshotter